apps/subscriber-app-e2e/src/integration/subscription-management.feature

```diff
@@ -43,3 +43,17 @@ Feature: Subscription management
     Then the user views a callout message of "Contact information updated."
     And the user views contact information of "auto.contact.test@gmail.com", "EMPTY" and "email"
 
+  @TEST_CS-1413 @REQ_CS-1240 @regression
```

Comment:
Could you please add a comment on what test data is needed for this test scenario? In case the user and its subscription are changed in the future causing this test to fail, we know what to be reset.

Reply:
comment added